base.yaml

```diff
@@ -131,7 +146,15 @@ management_portal:
     radar_upload_frontend: secret
     radar_rest_sources_backend: secret
     radarFitbitConnector: secret
+    morePromasysSync: secret
     radarAppconfig: secret
+  sendGrid:
```

Comment:
We don't need a reference to sendGrid, instead smtp can be used

Reply:
Wouldn't it be good to reference the configurable options in here?

Reply:
Yes that’s fine, but they’re just plain old SMTP settings, just rename sendGrid to smtp.

Reply:
Fixed in 71673c7